server/views/partials/header.njk

```diff
@@ -35,17 +35,12 @@
 </header>
 
 <div class="location-bar">
-  <span class="location-bar__location" data-test="active-location">
-    Hewell (HMP)
-  </span>
   {% if not hidePhaseBanner %}
-    {{ govukPhaseBanner ({
-      tag: {
-        text: "beta"
-      },
-       html: 'This is a new service, <a class="govuk-link" target="_blank" href="https://eu.surveymonkey.com/r/9ZBN3YJ">give us your feedback (opens in a new tab)</a>',
-      classes: 'bapv-phase-banner'
```

Comment:
If this is going, can we clean up the `.bapv-phase-banner` stuff from `local.scss`?

Reply:
@tpmcgowan Sorry only just saw this!

Reply:
@tpmcgowan PR raised https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/pull/402